### ISSUE TITLE
Video Indexer API ErrorType":"URL_UNREACHABLE" 

### ISSUE THREAD

**Issue by gregdegruy** [Azure/azure-storage-python#311]
Having issues uploading a vide file from an Azure blob using storage account name and key. Also tried adding a sas token using this guidance ()https://stackoverflow.com/questions/41285755/how-do-you-generate-the-signature-for-an-azure-blob-storage-sas-token-in-python

Still got this URL_UNREACHABLE" error
```
{"ErrorType":"URL_UNREACHABLE","Message":"Url 'https://myaccount.blob.core.windows.net/mycontainer/myblobpath/video.mp4?<i put my sas token here> is unreachable."}
```

Following these docs https://videobreakdown.portal.azure-api.net/docs/services/582074fb0dc56116504aed75/operations/582074fb0dc5610e14c75ec7

```python
########### Python 2.7 #############
import httplib, urllib, base64

headers = {
    # Request headers
    'Content-Type': 'multipart/form-data',
    'Ocp-Apim-Subscription-Key': '{subscription key}',
}

params = urllib.urlencode({
    # Request parameters
    'name': '{string}',
    'privacy': '{string}',
    'videoUrl': '{string}',
    'language': '{string}',
    'externalId': '{string}',
    'metadata': '{string}',
    'description': '{string}',
    'partition': '{string}',
    'callbackUrl': '{string}',
})

try:
    conn = httplib.HTTPSConnection('videobreakdown.azure-api.net')
    conn.request("POST", "/Breakdowns/Api/Partner/Breakdowns?%s" % params, "{body}", headers)
    response = conn.getresponse()
    data = response.read()
    print(data)
    conn.close()
except Exception as e:
    print("[Errno {0}] {1}".format(e.errno, e.strerror))
```

**Comment by pemari-msft** [comment]
Are you changing ``https://myaccount.blob.core.windows.net/mycontainer/myblobpath/video.mp4`` to reference your blob?

**Comment by gregdegruy** [comment]
Yes

**Comment by pemari-msft** [comment]
you should try accessing your blob through other means. If it can be reached, then it's not a storage issue. Unfortunately there's not much help we can provide at this point because we don't know what the video indexer is. Can you confirm the video is uploaded successfully and can be seen in other ways?

**Comment by gregdegruy** [comment]
Found the solution. I needed to include a sas token in my upload request.

This should be included in the docs in some way, how can I help include this in the docs?

```
# i use this little hack to get a properly formatted blob name first
blobName = block_blob_service.get_blob_to_path(IPutmycontainernameHere, IPutMyBlobPathAndNameHere, INameMyLocalFile).name

# then I make a sas token
sas_url = block_blob_service.generate_blob_shared_access_signature(IPutmycontainernameHere, blobName, BlobPermissions.READ, datetime.utcnow() + timedelta(hours=1))

# then I make a properly formatted blob URL
generated_blob_url = block_blob_service.make_blob_url(
    container_name= IPutmycontainernameHere,
    blob_name=blobName,
    sas_token=sas_url,
)
```

```
#-----------------------------------------
# Uploading Video to Video Indexer Service
#-----------------------------------------
headers = {
    'Content-Type': 'multipart/form-data',
    'Ocp-Apim-Subscription-Key': I_put_my_VIDEO_INDEXER_API_KEY_here,
}
params = urllib.urlencode({
    'name': 'NameOfVideo',
    'privacy': 'Public',
    'videoUrl': generated_blob_url
})
try:
    conn = httplib.HTTPSConnection('videobreakdown.azure-api.net')
    conn.request("POST", "/Breakdowns/Api/Partner/Breakdowns?%s" % params, "{body}", headers)
    response = conn.getresponse()
    data = response.read()
    print('Complete')
    print('Video ID: ' data)
    conn.close()
except Exception as e:
    print("[Errno {0}] {1}".format(e.errno, e.strerror))
```

**Comment by gregdegruy** [comment]
I found another error in the Account Videos tab of the `https://www.videoindexer.ai/?tab=account` page of the Video Indexer Portal as well. 

Any video uploaded programmatically from Python 2.7.11 as I've done, does not show in the portal. Any more context I can provide here? Is there some way I can send a screenshot. The video is viable in the portal by navigating to it using the video ID I printed to my command line `https://www.videoindexer.ai/media/videoIdHERE/`
